lib/cli/command/service/stage.rb

```diff
@@ -21,7 +21,7 @@ module Service
       subcommand_def 'stage' do |c|
         c.arg Token::Arg.assembly_name
         command_body c, :stage, 'Create a new service instance to refer to staged infrastructure that then can be deployed' do |sc|
-          sc.flag Token.module_ref, :desc => 'Module name with namespace from which to find assembly; not needed if command is executed from within the module'
+          sc.flag Token.directory_path, :desc => 'Absolute or relative path to service instance directory containing where updates will be pulled; not needed if in the service instance directory'
```

Comment:
This is an exception for a service command, but for stage since we arestaging from a module directory we need the path of the module directory if not doing dtk service stage inside of module directory. So Text should say:
:desc => 'Path to module directory where assembly is being staged from; not needed if in the module directory'